tssc/step_implementers/unit_test/junit.py

```diff
@@ -7,42 +7,53 @@
 Could come from either configuration file or
 from runtime configuration.
 
-| Configuration Key | Required? | Default | Description
-|-------------------|-----------|---------|-----------
-| `TODO`            | True      |         |
+| Configuration Key | Required? | Default     | Description
+|-------------------|-----------|-------------|-----------
+| `pom-file`        | True      | `'pom.xml'` | pom used to check for reportsDirectory definition
 
 Expected Previous Step Results
 ------------------------------
 
 Results expected from previous steps that this step requires.
 
-| Step Name | Result Key | Description
-|-----------|------------|------------
-| `TODO`    | `TODO`     | TODO
+None.
 
 Results
 -------
 
 Results output by this step.
 
-| Result Key | Description
-|------------|------------
-| `TODO`     | TODO
+| Result Key  | Description
+|-------------|------------
+| `junit`     | A dictionary of JUnit unit test result dictionaries
```

Comment:
shouldn't the result key name match the implementer name?

Reply:
no. like thats the parent key which you don't have contorl over. right now this is based on lines 157-162. im in a mixed bag if i like it that way. i guess it has to do with what we end up calling the class. If hte Class gets called `Maven` then it makes sense that the results has `junit` as a top level key because maybe later Maven can also output some other type of result or something. but i could also argue that its just uncessary and maybe the results should be:

```
results = {
  'test-results-dir': FODLER,
  'test-results-format': 'junit'
}
```

Reply:
Let's move this discussion to the broader conversation below on unit-test/junit maven/npm...